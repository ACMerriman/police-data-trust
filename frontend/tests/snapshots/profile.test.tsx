import * as React from "react"
import Profile from "../../pages/profile"
import renderer from "react-test-renderer"

it("renders Profile Page correctly", () => {
  const tree = renderer.create(<Profile />).toJSON()
  expect(tree).toMatchSnapshot()
})
