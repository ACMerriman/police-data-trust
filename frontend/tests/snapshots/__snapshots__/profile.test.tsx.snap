// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Profile Page correctly 1`] = `
<div>
  <div>
    <header
      class="wrapper"
    >
      <div
        class="backgroundBanner"
      >
        <div
          class="leftHeader"
        >
          <img
            alt="A blue circular logo design reads 'National Police Data Coalition'"
            class="logo medium"
            src="/NPDCLogo.svg"
          />
          <div
            class="titleContainer"
          >
            <h2>
              National Police Data Coalition
            </h2>
            <p>
              The national index of police incidents
            </p>
          </div>
        </div>
        <nav
          aria-label="Main Navigation"
        >
          <button
            aria-controls="navMenu"
            aria-expanded="false"
            class="dropdownTitle"
            type="button"
          >
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-ellipsis-v fa-w-6 fa-3x "
              data-icon="ellipsis-v"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 192 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M96 184c39.8 0 72 32.2 72 72s-32.2 72-72 72-72-32.2-72-72 32.2-72 72-72zM24 80c0 39.8 32.2 72 72 72s72-32.2 72-72S135.8 8 96 8 24 40.2 24 80zm0 352c0 39.8 32.2 72 72 72s72-32.2 72-72-32.2-72-72-72-72 32.2-72 72z"
                fill="currentColor"
              />
            </svg>
          </button>
          <ul
            id="navMenu"
          >
            <li
              class=""
            >
              <a
                href="/search"
              >
                Search
              </a>
            </li>
            <li
              class=""
            >
              <a
                href="/profile"
              >
                Profile
              </a>
            </li>
            <li
              class=""
            >
              <a
                href="/logout"
              >
                Sign Out
              </a>
            </li>
            <li
              class=""
            >
              <a
                href="#"
              >
                DONATE
              </a>
            </li>
          </ul>
          <button
            class="primaryButton"
            type="submit"
          >
            DONATE
          </button>
          <button
            class="primaryButton"
            type="submit"
          >
            LOGOUT
          </button>
        </nav>
      </div>
    </header>
    <div
      class="profileWrapper"
    >
      <nav
        class="leftMenu"
      >
        <ul>
          <li
            class="selectedItem"
          >
            User Information
          </li>
          <li
            class="menuItem"
          >
            Profile Type
          </li>
          <li
            class="menuItem"
          >
            Saved Results
          </li>
          <li
            class="menuItem"
          >
            Saved Searches
          </li>
        </ul>
      </nav>
    </div>
  </div>
</div>
`;
